Resources/Prototypes/GameRules/roundstart.yml

```
@@ -170,10 +170,8 @@
   components:
   - type: GameRule
     minPlayers: 5
-    delay:
-      min: 240
-      max: 420
   - type: AntagSelection
+    selectionTime: BeforeJobs # Goobstation
```

yamllint (1.35.1)

[warning] too few spaces before comment

(comments)